model/utils_test.go

```diff
@@ -350,16 +339,12 @@ func TestIsValidAlphaNum(t *testing.T) {
 
 	for _, tc := range cases {
 		actual := IsValidAlphaNum(tc.Input)
-		if actual != tc.Result {
-			t.Fatalf("case: %v\tshould returned: %#v", tc, tc.Result)
-		}
+		require.Equalf(t, actual, tc.Result, "case: %v\tshould returned: %#v", tc, tc.Result)
 	}
 }
 
 func TestGetServerIpAddress(t *testing.T) {
-	if len(GetServerIpAddress("")) == 0 {
-		t.Fatal("Should find local ip address")
-	}
+	require.NotEqual(t, len(GetServerIpAddress("")), 0, "Should find local ip address")
```

Comment:
`require.NotLen` here

Reply:
We do not have method for `require.NotLen` but we have it for `require.Len`. Or do we have?

Reply:
We can use `require.NotEmpty`.